src/include/cursor.h

```diff
@@ -587,6 +587,14 @@ struct __wt_cursor_version {
 #define WT_CURSOR_RAW_OK \
     (WT_CURSTD_DUMP_HEX | WT_CURSTD_DUMP_PRETTY | WT_CURSTD_DUMP_PRINT | WT_CURSTD_RAW)
 
+/*
+ * A cursor may or may not have bounds set. Checks if the cursor has lower or upper bounds set.
+ *
+ * This is primarily used by cursor bound checking logic.
```

Comment:
I think we can condense this comment to:
"This macro provides a consistent way of checking if a cursor has either its lower or upper bound set."